modules/storage-persistent-storage-iscsi-provisioning.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies
+//
+// * storage/persistent-storage-iscsi.adoc
+
+[[persistent-storage-iscsi-provisioning]]
+= Provisioning
+Verify that the storage exists in the underlying infrastructure before 
+mounting it as a volume in {product-title}. All that is required for the 
+iSCSI is the iSCSI target portal, a valid iSCSI Qualified Name (IQN), 
+a valid LUN number, the filesystem type, and the `PersistentVolume` API.
+
+Optionally, multipath portals and Challenge Handshake Authentication 
+Protocol (CHAP) configuration can be provided.
```

Comment:
Can you make this active tense? You can provide multipath portals and Challenge Handshake Authentication 
Protocol (CHAP) configuration ??

Reply:
I've adjusted this sentence; it now uses the recommendation after it's been slightly tweaked.